lib/yelp/responses/base.rb

```diff
@@ -0,0 +1,19 @@
+module Yelp
+  module Response
+    class Base
+      def initialize(json)
+        json.each do |key, value|
+          instance_variable_set("@#{key}", value)
+        end
+      end
+
+      private
+
+      def parse(json, klass)
+        return json.collect { |j| klass.new(j) } if json.class == Array
+        return klass.new(json) if json
```

Comment:
`klass` is a ruby convention.

https://github.com/rails/rails/blob/7f18ea14c893cb5c9f04d4fda9661126758332b5/activerecord/lib/active_record/aggregations.rb#L258